test/Microsoft.Azure.Devices.Edge.Test.Common/config/EdgeConfigBuilder.cs

```diff
@@ -18,9 +18,15 @@ public EdgeConfigBuilder(string deviceId)
             this.registries = new List<(string, string, string)>();
         }
 
-        public void AddRegistryCredentials(string address, string username, string password)
-        {
+        public void AddRegistryCredentials(string address, string username, string password) =>
             this.registries.Add((address, username, password));
+
+        public void AddRegistryCredentials(IEnumerable<(string address, string username, string password)> credentials)
```

Comment:
when did we need to pass more than 1 container registry credentials?

Reply:
When I was testing with 1.0.8-rc3 from our release builds ACR. I needed to get edgeAgent, edgeHub, and tempSensor from the release registry, but methodSender and methodReceiver aren't "released" so I still had to grab them from the daily builds registry.